src/utils/find-files.test.ts

```diff
@@ -0,0 +1,103 @@
+jest.mock('glob')
+jest.mock('path')
+jest.mock('./read-file')
+import glob from 'glob'
+import path from 'path'
+import { getFileExtensionPattern } from './read-file'
+const mockedGlob = glob as jest.Mocked<typeof glob>
+const mockedPath = path as jest.Mocked<typeof path>
+const mockedGetFileExtensionPattern = getFileExtensionPattern as jest.MockedFunction<
+  typeof getFileExtensionPattern
+>
+
+const mockedFileNames = ['/dev/config/development.yml', 'schema.json']
+const mockedResolvedPath = '/dev'
+mockedGlob.sync = jest.fn().mockReturnValue(mockedFileNames)
+mockedPath.resolve = jest.fn().mockReturnValue(mockedResolvedPath)
+mockedGetFileExtensionPattern.mockReturnValue('json')
+
+import { findConfigFiles, findFiles, isNotSchema } from './find-files'
+
+describe('findFiles works as expected', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('invokes glob.sync', () => {
```

Comment:
this test feels superfluous given that you also do `it('invokes glob.sync with correct default glob')`

Reply:
Was thinking that `glob.sync` can be invoked but wrong arguments are passed. In such cases, you can immediately see from the tests that 1. it's called and 2. but with wrong arguments.

Without that test, you would first have to debug if it's actually reaches `glob.sync` at all.

But nitty discussion but I still think this test _can_ be there.

Reply:
Hmm, but doesn't TypeScript protect us against calling this with wrong arguments by requiring them to be string?

Reply:
Not entirely sure. I think it's typesafe at transpile time but not at runtime.

Reply:
After some research, I found that Typescript only offers static analysis, so transpile-time type safety, by itself. With type guards and libraries such as `io-ts`, you can achieve basic runtime type-safety where needed.

I think it's important that we always validate user input, i.e. argument passed to `load()` and `validate()`, and data coming in programmatically (result of sops binary, results of KMS calls, etc.). Maybe worth another GitHub-issue to make a sanity check once we have all basic features in master.